RELEASES.md

```diff
@@ -375,6 +376,111 @@ See also the
 
+### Installing JAX Python packages
+
+Using the index pages [listed above](#installing-rocm-python-packages), you can
+also install `jaxlib`, `jax-rocm7-plugin`, and `jax-rocm7-pjrt`.
+
+> [!NOTE]
+> By default, pip will install the latest versions of each package. If you want to
+> install older versions take note of the compatibility matrix:
+>
+> | jaxlib version | jax-rocm7-plugin version | jax-rocm7-pjrt version |
+> | ------------- | ------------------ | ------------------- |
+> | 0.7.1         | 0.7.1               | 0.7.1              |
+>
+> For example, `jax` 0.7.1 and compatible wheels can be installed by specifying
+>
+> ```
+> jax-rocm7-plugin==0.7.1 jax-rocm7-pjrt==0.7.1 jaxlib==0.7.1
+> ```
+>
+> See also
+>
+> - [Supported Jax versions in TheRock](https://github.com/ROCm/TheRock/tree/main/external-builds/jax#supported-jax-versions)
+> - [Installing previous versions of Jax](https://github.com/ROCm/rocm-jax/releases/)
+
+> [!TIP]
+> The `jax` packages depend on `rocm[libraries]`, so ROCm packages should
+> be installed First.
+> [!NOTE]
+> We will update the installation of ROCm automalically soon just by installing jax wheels.
+
+
+#### jax for gfx94X-dcgpu
+
+Supported devices in this family:
+
+| Product Name  | GFX Target |
+| ------------- | ---------- |
+| MI300A/MI300X | gfx942     |
+
+```bash
+python -m pip install \
+  --index-url https://rocm.nightlies.amd.com/v2/gfx94X-dcgpu/ \
+  --pre jax-rocm7-pjrt jax-rocm7-plugin jaxlib
+```
+
+#### jax for gfx950-dcgpu
+
+Supported devices in this family:
+
+| Product Name  | GFX Target |
+| ------------- | ---------- |
+| MI350X/MI355X | gfx950     |
+
+```bash
+python -m pip install \
+  --index-url https://rocm.nightlies.amd.com/v2/gfx950-dcgpu/ \
+  --pre jax-rocm7-pjrt jax-rocm7-plugin jaxlib
+```
+
+#### jax for gfx110X-dgpu
+
+Supported devices in this family:
+
+| Product Name                       | GFX Target |
+| ---------------------------------- | ---------- |
+| AMD RX 7900 XTX                    | gfx1100    |
+| AMD RX 7800 XT                     | gfx1101    |
+| AMD RX 7700S / Framework Laptop 16 | gfx1102    |
+
+```bash
+python -m pip install \
+  --index-url https://rocm.nightlies.amd.com/v2/gfx110X-dgpu/ \
+  --pre jax-rocm7-pjrt jax-rocm7-plugin jaxlib
+```
+
+#### jax for gfx1151
+
+Supported devices in this family:
+
+| Product Name        | GFX Target |
+| ------------------- | ---------- |
+| AMD Strix Halo iGPU | gfx1151    |
+
+```bash
+python -m pip install \
+  --index-url https://rocm.nightlies.amd.com/v2/gfx1151/ \
+  --pre jax-rocm7-pjrt jax-rocm7-plugin jaxlib
+```
+
+#### jax for gfx120X-all
+
+Supported devices in this family:
+
+| Product Name     | GFX Target |
+| ---------------- | ---------- |
+| AMD RX 9060 / XT | gfx1200    |
+| AMD RX 9070 / XT | gfx1201    |
+
+```bash
+python -m pip install \
+  --index-url https://rocm.nightlies.amd.com/v2/gfx120X-all/ \
+  --pre jax-rocm7-pjrt jax-rocm7-plugin jaxlib
+```
```

Comment:
I don't see JAX packages for any of these GPU families.

For example https://rocm.nightlies.amd.com/v2/gfx110X-dgpu/ 

Workflows like https://github.com/ROCm/TheRock/actions/runs/19325285325/job/55274869858 seem to be uploading though... maybe missing from the root index file?